report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,38 +1,60 @@
+    \label{matrix:averaged_intensity}
+
+    \refstepcounter{listitem}
+    \item \textbf{Masked Intensity Matrix} $\;\mathbf{M}[\chi, l, \lambda]$: The result of $\mathbf{A}$ processed by applying wavelength-specific masks, setting intensities to zero in masking ranges to focus on relevant spectral features.
+    \label{matrix:masked_intensity}
 \end{itemize}
 
-The model outputs are the quantified chemical compositions of geological samples. These are primarily the concentrations of major elements, represented in weight percentage. While background elements are also present, our current analysis does not quantify these.
-Our goal is to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^O$, where $N$ represents the dimensionality of the processed \gls{libs} signals and $O$ represents the number of target oxides. This function maps a processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^O$ of estimated oxide concentrations. The vector $\mathbf{x}$ is derived from matrix $E$, representing processed spectral data:
+The model outputs are the quantified chemical compositions of geological samples. 
+These are primarily the concentrations of major elements, represented in weight percentage. 
+While background elements are also present, our current analysis does not quantify these. 
+Our goal is to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_o}$, where $N$ represents the dimensionality of the processed \gls{libs} signals and $n_o$ denotes the number of target oxides in the Concentration Matrix $\mathbf{C}[\chi, o]$.
+This function maps a processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^{n_o}$ of estimated oxide concentrations. The vector $\mathbf{x}$ is derived from matrix $\mathbf{M}$, representing processed spectral data:
 \[
 \mathbf{v} = \mathcal{F}(\mathbf{x}).
 \]
 
+As previously introduced, quantifying chemical compositions from \gls{libs} spectral data poses significant challenges, including high dimensionality and multicollinearity of the data, as well as matrix effects. Here, we further examine these challenges:
+
+\begin{itemize}
+    \item \textbf{Data Dimensionality and Collinearity:} The high dimensionality of spectral data, coupled with multicollinearity—where multiple spectral features exhibit strong correlations—complicates the modeling and analysis. 
+    This challenge arises both in the Intensity Matrix $\mathbf{I[\chi, l, s, \lambda]}$ and consequently the Averaged Intensity Matrix $\mathbf{A[\chi, l, \lambda]}$, where the high number of wavelengths leads to a large number of dimensions, and the intensity readings across these wavelengths often display high correlations.
```

Comment:
To be more precise, I would say it's more evident from $\lambda>\chi$ in all cases we know of. That is, there are more dimensions than samples.

Reply:
And I think we might want to separate this kind of sentence into two sentences. First, we clarify the challenge of dimensionality and how it affects the data. Then we do the same for collinearity.

Reply:
```suggestion
    \item \textbf{Data Dimensionality and Collinearity:} The high dimensionality of spectral data complicates the modeling and analysis. This issue is evident because there are more wavelengths ($\lambda$) than samples ($\chi$), leading to a greater number of dimensions than samples. Consequently, this arises both in the Intensity Matrix $\mathbf{I[\chi, l, s, \lambda]}$ and the Averaged Intensity Matrix $\mathbf{A[\chi, l, \lambda]}$. Additionally, multicollinearity, where multiple spectral features exhibit strong correlations, further complicates the analysis as the intensity readings across these wavelengths often display high correlations.
```
Something like that?